postgres/src/test/java/org/jdbi/v3/TestStatementsTimeout.java

```diff
@@ -27,7 +28,18 @@
 
 public class TestStatementsTimeout {
     @Rule
-    public JdbiRule dbRule = PostgresDbRule.rule();
+    public JdbiRule dbRule = PostgresDbRule.rule(builder -> {
+        // We need to force the locale for the 'testTimeout' test
+        final String locale;
+
+        if (SystemUtils.IS_OS_WINDOWS) {
+            locale = "English_United States";
```

Comment:
Is there a constant here we could use that doesn't vary by platform?  Like `Locale.ROOT`?

Reply:
We could also fix it to a constant for every test case in the `JdbiRule` factory.

Reply:
Yeah I thought about just changing JdbiRule to do this but realized this would force all users of the library to also have their tests be forced to en_US or whatever default we choose (which should be fine for most people, but will suck for the couple of edge cases where you do need another locale). I could add a `JdbiRule::embeddedEnUsPostgres()` function that takes care of setting this when required but would still let users use whatever locale they want.

Reply:
It looks like the locale is only relevant because we look at the error message.  We could check the SQLSTATE of the exception instead and then we don't depend on locale at all, perhaps?

Reply:
Good call, I've removed the locale dependency on this test.